pkg/ovs/openflow/ofctrl_packetout.go

```diff
@@ -319,6 +332,9 @@ func (b *ofPacketOutBuilder) Done() *ofctrl.PacketOut {
 		klog.Errorf("Invalid PacketOutBuilder: IP header and IPv6 header are not allowed to exist at the same time")
 		return nil
 	}
+	if b.pktOut.InPort == 0 {
+		b.pktOut.InPort = openflow15.P_CONTROLLER
```

Comment:
Doesn't ofnet set it when it's 0?

Reply:
Originally ofnet does not have set default value in the case `Inport` is 0. So I added the logic in antrea code. But with the final code, ofnet uses `openflow15.P_CONTROLLER` in this case. I removed this code in antrea with my latest update.